.preview/build/webpack_config.js

```diff
@@ -17,7 +17,7 @@ const {
   ASSET_REGEX,
   SVG_REGEX,
   SCRIPT_REGEX,
-} = require('./webpack-base');
+} = require('./webpack_base');
```

Comment:
I'm wondering, what is the change in here? :D

Reply:
We updated some linter packages and somehow there is a rule about file naming